CodeStarterCamp_Week2/MyLottoNumber.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  MyLottoNumber.swift
+//  CodeStarterCamp_Week2
+//
+//  Created by 황지웅 on 2022/04/21.
+//
+
+import Foundation
+
+let lottoCountLimit = 6
+let lottoNumberLimit = 45
+let lottoNumberRange = 1...lottoNumberLimit
+let lottoNumberSet = [Int](1...45)
```

Comment:
```suggestion
let lottoCountLimit = 6
let lottoNumberLimit = 45
let lottoNumberRange = 1...lottoNumberLimit
let lottoNumberSet = [Int](lottoNumberRange)
```
사용하지 않는 변수가 있네요! 위에처럼 쓰려고 했던 것일까요..?